src/run_bpftrace.cpp

```diff
@@ -81,8 +81,10 @@ int run_bpftrace(BPFtrace &bpftrace,
 
   std::cout << "\n\n";
 
+  bpftrace.print_benchmark_results(output);
```

Comment:
I know this won't print anything if there are no benchmark results but maybe it reads easier if it's gated on `run_benchmarks` then you can just do
```
if (bpftrace.run_benchmarks_) {
  bpftrace.print_benchmark_results(output);
} else if (bpftrace.config_->print_maps_on_exit) {
  err = bpftrace.print_maps(output);
}
```